/drive/model_trainer_node:
  ros__parameters:
    path_to_input_space_calib: "/home/nicolassamson/ros2_ws/src/DRIVE/calib_data/test_warthog_reassemble/input_space_data.pkl"
    path_to_calibration_node_config_file: "/home/nicolassamson/ros2_ws/src/DRIVE/calib_data/test_warthog_reassemble/config_file_used/_warthog.config.yaml"
    path_to_calibration_data_raw: "/home/nicolassamson/ros2_ws/src/DRIVE/calib_data/test_warthog_reassemble/model_training_datasets/data_raw.pkl"
    run_by_maestro: False
    imu_inverted: False
    training_horizon: 2.0
    pwrtrain_model_config_path: "/home/nicolassamson/ros2_ws/src/DRIVE/motion_model_available/_pwrtain_motion_model_parameters.yaml"
    model_results_path: "/home/nicolassamson/ros2_ws/src/DRIVE/calib_data/test_warthog_reassemble/model_training_results"
    slip_BLR_model_config_path: "/home/nicolassamson/ros2_ws/src/DRIVE/motion_model_available/_slip-BLR_motion_model_parameters.yaml"